packages/crayons-core/src/components/options-list/list-options.tsx

```diff
@@ -55,6 +55,14 @@ export class ListOptions {
    * Place a checkbox.
    */
   @Prop() isCheckbox = false;
+  /**
+   * Default option to be shown if the option doesn't match the filterText.
+   */
+  @Prop() notFoundText = 'No items Found';
+  /**
+   * Placeholder to placed on the search text box.
+   */
+  @Prop() searchText = 'Search...';
```

Comment:
Let's remove the "..."

Reply:
The test _Search..._ is based on the DSM spec.